### DISCUSSION TITLE
keras 3, composable ml systems, croissant

### DISCUSSION THREAD

**Discussion by luisoala** [keras-team/keras#19715] [General]
hi @haifeng-jin it was nice running into you today at mlsys

@hertschuh haifeng mentioned you might be a good person to drag into this exchange (:

we have been working on a framework-agnostic format for ml-ready data, called croissant

so far it is integrated by hf, kaggle and openml on ~400k public datasets

id be curious to have a short chat how this gels with keras 3.0, in particular the framework agnostic data loaders

let me know if you are interested to discuss this over a coffee irl or virtually

mini intro paper to croissant: https://arxiv.org/abs/2403.19546
code: https://github.com/mlcommons/croissant

![image](https://github.com/keras-team/keras/assets/26168435/412baca0-2cb7-4317-af5c-e9a686eb9c13)

**Comment by hertschuh** [comment]
Hi @luisoala , this sounds interesting. Where are you located? (I'm not at mlsys)

**Reply by luisoala** [reply]
im in the bay area this week and next, afterwards in switzerland. if any of these work for you we can meet in person. else have a quick call (:

**Comment by hertschuh** [comment]
I'm in the bay area today and tomorrow. But it might be easier to have a call.

**Reply by luisoala** [reply]
tonight i could do dinner after 6. tomorrow i could do lunch between 11.30 and 1. if no good we find time for call (:

**Reply by hertschuh** [reply]
How can I contact you directly? I'm between Sunnyvale and Mountain View. Is that close to where you are in the bay area?

**Reply by luisoala** [reply]
yes perfect also in sunnyvale

i added you to a private repo with contact coordinates (:
